tests/data/profile_stability/rhel8/stig.profile

```diff
@@ -207,10 +207,9 @@ selections:
 - enable_authselect
 - enable_dracut_fips_module
 - enable_fips_mode
+- enable_gpgcheck_for_all_repositories
 - encrypt_partitions
-- ensure_gpgcheck_globally_activated
```

Comment:
Do these need to go into the `default.profile`?

Reply:
They don't, they are part of many other RHEL 8 profiles (CIS, ANSSI, ...).